modules/apps/headless/headless-admin-site/headless-admin-site-impl/src/main/java/com/liferay/headless/admin/site/internal/resource/v1_0/SiteResourceImpl.java

```diff
@@ -262,12 +234,14 @@ public Site postSiteSiteInitializer(MultipartBody multipartBody)
 
 		Site site = multipartBody.getValueAsInstance("site", Site.class);
 
-		return putSiteByExternalReferenceCode(
+		return putSiteSiteInitializer(
 			site.getExternalReferenceCode(), multipartBody);
 	}
 
 	@Override
-	public Site putSite(Site site) throws Exception {
+	public Site putSite(String siteExternalReferenceCode, Site site)
```

Comment:
The `siteExternalReferenceCode` is unused. How do other APIs use the supplied ERC versus the ERC within the actual object?